src/sage/rings/polynomial/multi_polynomial_libsingular.pyx

```diff
@@ -5744,6 +5744,78 @@ cdef class MPolynomial_libsingular(MPolynomial_libsingular_base):
         else:
             return self * self.denominator()
 
+    def in_subalgebra(self, J, algorithm=None):
+        """
+        Return whether this polynomial is contained in the subalgebra
+        generated by ``J``
+
+        INPUT:
+
+        - ``J`` -- iterable of elements of the parent polynomial ring
+
+        - ``algorithm`` -- can be ``None`` (the default), "algebra_containment",
+          "inSubring", or "groebner".
+
+          - ``None`` (the default): use "algebra_containment".
+
+          - "algebra_containment": use Singular's ``algebra_containment`` function,
+            https://www.singular.uni-kl.de/Manual/4-2-1/sing_1247.htm#SEC1328. The
```

Comment:
Why not just 
```
- ``'algebra_containment'``: (default) uses Singular's ``algebra_containment`` function ...
```
Then you can get rid of `None``, which is really not an option.

Reply:
Good idea, I'll do that.

Reply:
Thanks.